AI/MIGraphX/Quantization/Running Quantized ResNet50 via MIGraphX.md

```diff
@@ -0,0 +1,105 @@
+
+
+# Running quantized ResNet50 via MIGraphX
+
+## Summary
+This example walks through the dynamo Post Training Quantization (PTQ) workflow for running a quantized model using torch_migraphx.
+
+## Prerequisites
+
+- You must follow the installation instructions for the torch_migraphx library in [Readme.md](https://github.com/Rmalavally/rocm-examples/blob/develop/AI/MIGraphX/Quantization/Readme.md) before using this example.
+
+## Steps for running a quantized model using torch_migraphx
+
+1. Use torch.export and quantize_pt2e APIs to perform quantization
+*Note*: The export API call is considered a prototype feature at the time this tutorial is written. Some call signatures may be modified in the future.
+
+```import torch
+from torchvision import models
+from torch._export import capture_pre_autograd_graph
+from torch.ao.quantization.quantize_pt2e import prepare_pt2e, convert_pt2e
+```
+
+```import torch_migraphx
+from torch_migraphx.dynamo.quantization import MGXQuantizer
+model_fp32 = models.resnet50(weights=models.ResNet50_Weights.IMAGENET1K_V1).eval()
+input_fp32 = torch.randn(2, 3, 28, 28)
+
+torch_fp32_out = model_fp32(input_fp32)
+```
+
+The capture_pre_autograd_graph call will be changed to a torch.export.export call once it supports the pre autograd capture functionallity. 
```

Comment:
This can be removed, it will probably just confuse the user. We will need to update this example when/if PyTorch makes this change.

Reply:
Fixed